discord/client.py

```diff
@@ -1153,3 +1153,115 @@ def transform(d):
+            Whether or not to automatically convert emoticons into emojis.
+            e.g. :-) -> 😃
+        default_guilds_restricted: Optional[:class:`bool`]
+            Whether or not to automatically disable DMs between you and
+            members of new guilds you join.
+        detect_platform_accounts: Optional[:class:`bool`]
+            Whether or not to automatically detect accounts from services
+            like Steam and Blizzard when you open the Discord client.
+        developer_mode: Optional[:class:`bool`]
+            Whether or not to enable developer mode.
+        disable_games_tab: Optional[:class:`bool`]
+            Whether or not to disable the showing of the Games tab.
+        enable_tts_command: Optional[:class:`bool`]
+            Whether or not to allow tts messages to be played/sent.
+        explicit_content_filter: Optional[:class:`UserContentFilter`]
+            The filter for explicit content in all messages.
+        friend_source_flags: Optional[:class:`FriendFlags`]
+            Who can add you as a friend.
+        gif_auto_play: Optional[:class:`bool`]
+            Whether or not to automatically play gifs that are in the chat.
+        guild_positions: Optional[List[:class:`int`]]
+            A list of guild ids in order of the guild/guild icons that are on
+            the left hand side of the UI.
+        inline_attachment_media: Optional[:class:`bool`]
+            Whether or not to display attachments when they are uploaded in chat.
+        inline_embed_media: Optional[:class:`bool`]
+            Whether or not to display videos and images from links posted in chat.
+        locale: Optional[:class:`str`]
+            The RFC 3066 language identifier of the locale to use for the language
+            of the Discord client.
+        message_display_compact: Optional[:class:`bool`]
+            Whether or not to use the compact Discord display mode.
+        render_embeds: Optional[:class:`bool`]
+            Whether or not to render embeds that are sent in the chat.
+        render_reactions: Optional[:class:`bool`]
+            Whether or not to render reactions that are added to messages.
+        restricted_guilds: Optional[List[:class:`int`]]
+            A list of guild ids that you will not receive DMs from.
+        show_current_game: Optional[:class:`bool`]
+            Whether or not to display the game that you are currently playing.
+        status: Optional[:class:`Status`]
+            The clients status that is shown to others.
+        theme: Optional[:class:`Theme`]
+            The theme of the Discord UI.
+        timezone_offset: Optional[:class:`int`]
+            The timezone offset to use.
+
+        Raises
+        -------
+        HTTPException
+            Editing the settings failed.
+        Forbidden
+            The client is a bot user and not a user account.
+
+        Returns
+        -------
+        :class:`dict`
```

Comment:
This return type feels lazy but I don't really see much point in wrapping this into a class since this is only for user bots.

Consider this a passing comment.